web/styles/portico/comparison_table.css

```diff
@@ -228,11 +226,6 @@
         letter-spacing: 0.1ch;
     }
 
-    .comparison-table th.sticky {
-        padding: 8px 2px;
-        border-radius: 0;
-    }
```

Comment:
This change seems not like the rest of this commit?

Reply:
Ah yes, squashing mistake—moved this to its own commit.